### ISSUE TITLE
Theater Mode like Twitch.

### ISSUE THREAD

**Issue by Bubulae** [YouTube-Enhancer/extension#668]
I think its been months since youtube streaming change to match up with twitch, where the the live chat will show on the right side when on theater mode.

But the screen still not as good as twitch, because on twitch the screen "fit" perfectly and looks clean, while on youtube it doenst fit.

Would be nice if the screen on theater mode (feature) can bit fit just like how theater mode on twitch.

Thanks for reading this..

**Comment by VampireChicken12** [comment]
@Bubulae Thank you for suggesting this feature. I think it should be possible can you provide more details on how you think this could be implemented to fit your needs. 

I compared YouTube and Twitch's theater mode on live videos and what I gather is that you'd like the video and the chat to fill the height of the window like it does on Twitch. Correct me if I am wrong.

**Comment by Bubulae** [comment]
Thank you for replying. Sorry if my suggestion not really specific, but what I meant by that, you can see the different between Youtube vs Twitch Theater mode.

Youtube :

![youtubes](https://github.com/user-attachments/assets/222fd5b3-b03c-4307-bc8c-d725e6b8be44)

Twitch : 

![image](https://github.com/user-attachments/assets/4492a7a4-cb5b-42db-bc92-088031f315a2)

As you can see, twitch did it better because the the theater mode "fit" the screen entirely. So it looks cleans if I want to watch the stream with livechat on. And it even better if I go full screen (F11),

Youtube the other hand, the video not entirely "fit" the screen. We still can see the stream title, like/dislike, desc, vid rec, etc below it.. Not so clean especially if we plan to watch this on full screen (F11) with livechat on. 

Would be great if this extension have that option where to make the video screen fit the screen so give the clean look with livechat while watching streaming.

Thanks for reading this..

**Comment by VampireChicken12** [comment]
Thank you for clarifying. I will look into implementing this feature. No estimate on when it will be released. When the feature is added I'll upload the extension for loading in unpacked form as a temporary solution.

**Comment by VampireChicken12** [comment]
@shrublet Thanks for commenting, Can you make a issue for this.

**Comment by VampireChicken12** [comment]
Also can you provide a video showing the issue that you are talking about occurring. Also can you show the intended function

**Comment by NoamVH** [comment]
Hello,

Anything new with this one?

Ever since I started using Enhancer the chat stopped being docked to the right when I change to theater mode, and instead it just goes down like before YouTube added that feature, which is currently missing for me.

The feature requrested here won't only return chat docking in theater mode, but will make it better than what YouTube is currently offering.

I came here to request that feature, but I found this when searching to prevent a duplication.

**Comment by VampireChicken12** [comment]
@NoamVH Sorry I haven't looked into adding the feature requested in this issue although I suspect you are using Enhancer for YouTube which is a different extension then the one this repository is related to.

**Comment by VampireChicken12** [comment]
> Somewhat unrelated, but I just started using this extension, and currently, it breaks YouTube's default theater mode behavior with livestreams where the chat is placed on the side. Instead, the livestream fills the entire horizontal space like a typical video. I tried disabling all features that could affect this, but to no avail.

The person who commented this was using the other extension that I have no relation to. 

#699

**Comment by NoamVH** [comment]
> [@NoamVH](https://github.com/NoamVH) Sorry I haven't looked into adding the feature requested in this issue although I suspect you are using Enhancer for YouTube which is a different extension then the one this repository is related to.

You are correct, I am terribly sorry for my mistake, I'll look for the correct repository.

**Comment by VampireChicken12** [comment]
> > [@NoamVH](https://github.com/NoamVH) Sorry I haven't looked into adding the feature requested in this issue although I suspect you are using Enhancer for YouTube which is a different extension then the one this repository is related to.
> 
> You are correct, I am terribly sorry for my mistake, I'll look for the correct repository.

It's okay no worries
